src/components/utils/link.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+/**
+ * Link component props.
+ *
+ * @export
+ * @param {object} props
+ * @param {string} props.url
+ * @param {boolean} props.newTab
+ * @returns {ReactNode}
+ */
+const Link = ({ href, children, newTab }) => {
```

Comment:
You could consider following the convention we talked about earlier about clear usage of booleans and call the prop: "shouldOpenNewTab"

Reply:
nice spotted! I named it isNewTab, for ease of use :)